lib/client-options.js

```diff
@@ -44,7 +44,8 @@ function defaultOptions () {
     authProvider: null,
     maxPrepared: 500,
     encoding: {
-      copyBuffer: true
+      copyBuffer: true,
+      useUndefinedAsUnset: true
```

Comment:
`useUndefinedAsUnset` only affects protocol v4 and above, when using unset or `undefined` for protocol version < 4, [it will throw a TypeError](https://github.com/datastax/nodejs-driver/blob/9f83ddbf411f2d568f6b9c780d1fef71ffd1a3cb/lib/encoder.js#L179).